airflow/providers/mysql/CHANGELOG.rst

```diff
@@ -26,6 +26,12 @@
 Changelog
 ---------
 
+5.3.0
+.....
```

Comment:
lets not set it in advanced. I prefer the method of setting the version only when we know it for sure (major release) for all other cases just keep entries at the top is easier to pick them up

```suggestion
5.3.0
.....
```

Reply:
Aye Aye. This is what i've been always doing before, but I thought we changed it :)